lib/proxyquire.js

```diff
@@ -96,6 +96,19 @@ Proxyquire.prototype.preserveCache = function () {
   return this.fn
 }
 
+/**
+ * Stubbing a module that does not exist (from the perspective of the proxyquire
+ * subject) will throw an error.
+ * @name throwOnUnresolved
+ * @function
+ * @private
+ * @return {object} The proxyquire function to allow chaining
+ */
+Proxyquire.prototype.throwOnUnresolved = function () {
```

Comment:
`throwOnNotFound` is the best I can come up with. Handling unresolved stubs is more of a description of how proxyquire works as opposed to what the user wants.